components/layout_2020/style_ext.rs

```diff
@@ -420,8 +420,7 @@ impl ComputedValuesExt for ComputedValues {
         //     elements with the exception of any descendant element of text content
         //     elements."
         // https://drafts.csswg.org/css-transforms/#transformable-element
-        // FIXME(mrobinson): Properly handle tables and replaced elements here.
-        if self.get_box().display.is_inline_flow() {
+        if self.get_box().display.outside() == stylo::DisplayOutside::Inline && !is_replaced {
```

Comment:
I don't think this should return false for `inline-table`, `inline-block` or `inline-flex`. You should keep `is_inline_flow()`.